sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -661,6 +664,15 @@ private object CosmosReadConfig {
       "entry."
   )
 
+  private val RuntimeFilteringEnabled = CosmosConfigEntry[Boolean](
+      key = CosmosConfigNames.ReadRuntimeFilteringEnabled,
+      mandatory = false,
+      defaultValue = Some(false),
```

Comment:
Should default be true?

Reply:
I think previously, I am a little bit overload the runtime filtering usage. 
I have separate the runtime filtering and the readMany filtering config.  runtime filtering will be enabled automatically, however for readMany filtering, i think we should keep the default as false as it may added extra _itemIdentity column and it could be breaking changes for some customers.

Reply:
Agreed - makes sense. Also, when there are id/pkvalue predictates + others you might sometimes want to stick using query

Reply:
LGTM